runtime/binding/python/py/inference.py

```diff
@@ -80,13 +80,29 @@ def extract_embedding_wav(self, wav_path: str, resample_rate: int = 16000):
         Args:
             wav_path: the path of wav
             resample_rate: sampling rate
+        Return:
+            embeddings(list): [1, emb_dim]
         """
         feats = self._compute_fbank(wav_path, resample_rate=resample_rate)
         feats = np.expand_dims(feats, 0)
         embeddings = self.session.run(output_names=['embs'],
                                       input_feed={'feats': feats})
         return embeddings[0]  # [1, emb_dim]
 
+    def extract_embedding_feat(self, feats):
+        """ Extract embedding from feat(fbank).
+        Args:
+            feats(numpy.ndarray): [B, T, D]
+        Returns:
+            embeddings(list): [B, emb_dim]
+        """
+        assert isinstance(
+            feats, np.ndarray), 'NOTE: the type of feats need be np.ndarray.'
+        assert len(feats.shape) == 3, "NOTE: the shape of feats is [B, T, D]."
+        embeddings = self.session.run(output_names=['embs'],
+                                      input_feed={'feats': feats})
+        return embeddings[0]  # [B, embed]
+
     def extract_embedding(self,
```

Comment:
This is the most important interface. I think we should keep it for extracting embedding and return numpy ndarrary, which is preferred for most users.

And we can add something like `extract_embedding_kaldi`, in which the input and output are in kaldi style(scp/ark)